oidc-service-impl/src/main/java/io/mosip/esignet/services/TokenServiceImpl.java

```diff
@@ -122,6 +124,7 @@ public String getAccessToken(OIDCTransaction transaction, String cNonce) {
         payload.put(CLIENT_ID, transaction.getClientId());
 
         if(cNonce != null) {
+            payload.put(AUD, scopesResourceMapping.getOrDefault(transaction.getPermittedScopes().get(0), ""));
```

Comment:
are we not taking of the situation of multiple credential scope, where we may need to put the multiple resource here ?

Reply:
Currently, we are only considering a single scope. Since there are some unknowns and clarifications required with multiple resources support